From 5e55de93f8be08e77d6bcaaa8c3fc5911b628ebe Mon Sep 17 00:00:00 2001
From: Meticulus <theonejohnnyd@gmail.com>
Date: Wed, 19 Mar 2014 15:28:43 -0500
Subject: [PATCH] Implement BOARD_USES_STE_HARDWARE [2/2]

-Switches build to include Flattenable.cpp in system/core/libutils

-Keeps source clean for other targets.

Change-Id: Ided4ae326b88190c1f0142250f668514dce2e662
---
 libutils/Android.mk | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/libutils/Android.mk b/libutils/Android.mk
index d48991d..02e16b8 100644
--- a/libutils/Android.mk
+++ b/libutils/Android.mk
@@ -22,7 +22,6 @@ commonSources:= \
 	BlobCache.cpp \
 	CallStack.cpp \
 	FileMap.cpp \
-	Flattenable.cpp \
 	JenkinsHash.cpp \
 	LinearAllocator.cpp \
 	LinearTransform.cpp \
@@ -44,6 +43,10 @@ commonSources:= \
 	VectorImpl.cpp \
 	misc.cpp
 
+ifeq ($(BOARD_USES_STE_HARDWARE),true)
+	commonSources += Flattenable.cpp
+endif
+
 host_commonCflags := -DLIBUTILS_NATIVE=1 $(TOOL_CFLAGS)
 
 ifeq ($(HOST_OS),windows)
-- 
1.9.3

